src/python_testing/TC_ACE_1_3.py

```diff
@@ -46,6 +46,72 @@ async def read_descriptor_expect_unsupported_access(self, th):
         await self.read_single_attribute_expect_error(
             dev_ctrl=th, endpoint=0, cluster=cluster, attribute=attribute, error=Status.UnsupportedAccess)
 
+    def desc_TC_ACE_1_3(self) -> str:
+        return ""
```

Comment:
```suggestion
        return "TC_ACE_1_3"
```
In order to present the TC name in TH UI,m we need this method filled with the test case name